src/Sylius/Component/Core/Model/Order.php

```diff
@@ -318,6 +318,20 @@ public function getLastPayment($state = null)
     }
 
     /**
+     * @return bool
+     */
+    public function requiresShipping()
```

Comment:
Can do, provided everyone is happy - just took the name suggested in #8380, easy enough to change!